sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -197,6 +197,11 @@ class device_global {
 
 _T_ is restricted to types that have a trivial destructor. _PropertyListT_ enables properties to be associated with a `device_global`.
 
+[NOTE]
+====
+If _T_ is `const` then implementations may choose to allocate the `device_global` in a dedicated constant address space as an optimization. When using the {dpcpp} compiler with the CUDA or HIP backend, declaring a `device_global<const T>` is equivalent to declaring a `$$__constant__$$` variable.
+====
```

Comment:
This doesn't seem like it corresponds to what is implemented. It _could_ be done in tandem by making all conditional behavior dependent on the new property dependent on the disjunction of that and `std::is_const_v<T>`. That said, if that is the case we also need to document how that affects the members of the `device_global` class.

Reply:
I haven't finalised the implementation yet (I'm just testing the draft requested changes atm), so I haven't updated the documentation which you are right is completely out of date. I described this here: https://github.com/intel/llvm/pull/16001#discussion_r1879841363

Reply:
Can you mention me with a comment like "spec ready for review" when it is ready?

Reply:
> Can you mention me with a comment like "spec ready for review" when it is ready?

Sure, no problem.